tests/functional/spec/ukis/routing/why-no-innovation.spec.js

```diff
@@ -0,0 +1,473 @@
+  import InnovationsProtectedPatents from '../../../pages/surveys/ukis/innovations-protected-patents.page.js'
+  import InnovationsProtectedDesign from '../../../pages/surveys/ukis/innovations-protected-design.page.js'
+  import InnovationsProtectedCopyright from '../../../pages/surveys/ukis/innovations-protected-copyright.page.js'
+  import InnovationsProtectedTrademark from '../../../pages/surveys/ukis/innovations-protected-trademark.page.js'
+  import InnovationsProtectedLeadTime from '../../../pages/surveys/ukis/innovations-protected-lead-time.page.js'
+  import InnovationsProtectedServices from '../../../pages/surveys/ukis/innovations-protected-services.page.js'
+  import InnovationsProtectedSecrecy from '../../../pages/surveys/ukis/innovations-protected-secrecy.page.js'
+  import CoOperationOnInnovationCompleted from '../../../pages/surveys/ukis/co-operation-on-innovation-completed.page.js'
+  import PublicFinancialSupport from '../../../pages/surveys/ukis/public-financial-support.page.js'
+  import KindFinancialCentralGovernmentSupport from '../../../pages/surveys/ukis/kind-financial-central-government-support.page.js'
+  import PublicFinancialSupportForInnovationCompleted from '../../../pages/surveys/ukis/public-financial-support-for-innovation-completed.page.js'
+  import Turnover2014 from '../../../pages/surveys/ukis/turnover-2014.page.js'
+  import Turnover2016 from '../../../pages/surveys/ukis/turnover-2016.page.js'
+  import Exports2016 from '../../../pages/surveys/ukis/exports-2016.page.js'
+  import TurnoverAndExportsCompleted from '../../../pages/surveys/ukis/turnover-and-exports-completed.page.js'
+  import Employees2014 from '../../../pages/surveys/ukis/employees-2014.page.js'
+  import Employees2016 from '../../../pages/surveys/ukis/employees-2016.page.js'
+  import EmployeesQualifications2016 from '../../../pages/surveys/ukis/employees-qualifications-2016.page.js'
+  import EmployeesInHouseSkills from '../../../pages/surveys/ukis/employees-in-house-skills.page.js'
+  import EmployeesAndSkillsCompleted from '../../../pages/surveys/ukis/employees-and-skills-completed.page.js'
+  import AdditionalComments from '../../../pages/surveys/ukis/additional-comments.page.js'
+  import HowLong from '../../../pages/surveys/ukis/how-long.page.js'
+  import ApproachedTelephone from '../../../pages/surveys/ukis/approached-telephone.page.js'
+  import ReadyToSubmitCompleted from '../../../pages/surveys/ukis/ready-to-submit-completed.page.js'
+  import Navigation from '../../../pages/surveys/ukis/navigation.page.js'
+
+  const expect = chai.expect
+
+  describe('UKIS - Why no innovation?', function() {
+
+    it('Given I am answering question 6.13 under 3. Constrains of Innovation, When I have selected yes to ANY of questions 2.1, 3.1, 3.4, 3.6, 3.9, 3.11, 3.13, 3.15, 4.1, 4.3, 5.1 6.1 , I will NOT see question 6.14 and will be routed to the interstitial page at the end of section 6', function() {
```

Comment:
The when I have select could be simplified to `when I have select yes` and the bloated repeated text could be put in the description